Dockerfile

```diff
@@ -1,14 +1,28 @@
+# syntax=docker/dockerfile:1
 FROM golang:1.18 AS build
 
+WORKDIR /app
+COPY . .
 RUN apt-get update -y && apt-get install -y xz-utils
-RUN useradd -u 1000 -m esm
-RUN mkdir /esm && chown esm:esm /esm
-RUN git clone https://github.com/esm-dev/esm.sh /esm/esm.sh
-RUN git checkout v124
```

Comment:
again, let's use the latest tag instead of main branch

Reply:
https://github.com/gocrane/crane/blob/9776243d8f37f2766fa7f39ffb4b2470de922857/.github/workflows/build-images.yml#L90C12-L95

Reading the version information from git is better than manual changes each time.